src/main/java/com/coreos/jetcd/data/Pair.java

```diff
@@ -0,0 +1,34 @@
+package com.coreos.jetcd.data;
```

Comment:
agreed. pair is merely a helper structure. I probaly have another pkg called `com.coreos.jetcd.internal` and put pair there?